### PR TITLE
Prep for v0.1.0 release

### PR DESCRIPTION
**Pull request by nturinski**
- [x] Bump version
- [x] Update changelog
- [x] Update README (if necessary)
- [x] Make sure package-lock.json is up to date
- [x] Generate current NOTICE.html file (See Using third party code) and merge into repo

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1 +1,15 @@
+<!-- markdownlint-disable MD024 -->
```
Comment (wwlorey) [inline CHANGELOG.md:1]: Why is this needed?

File: README.md [hunk 2]
```diff
@@ -1,33 +1,53 @@
-# Project
+# Azure Container Apps for Visual Studio Code
 
-> This repo has been populated by an initial template to help get you started. Please
-> make sure to update the content to build a great experience for community-building.
+<!-- region exclude-from-marketplace -->
 
-As the maintainer of this project, please make a few updates:
+[![Version](https://vsmarketplacebadge.apphb.com/version/ms-azuretools.vscode-azurecontainerapps.svg)](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurecontainerapps) [![Installs](https://vsmarketplacebadge.apphb.com/installs-short/ms-azuretools.vscode-azurecontainerapps.svg)](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurecontainerapps) [![Build Status](https://dev.azure.com/ms-azuretools/AzCode/_apis/build/status/vscode-azurecontainerapps?branchName=main)](https://dev.azure.com/ms-azuretools/AzCode/_build?definitionId=39&branchName=main)
```
Comment (wwlorey) [inline README.md:5]: The build status shows as not set up yet. Will that populate once this PR merges or is there a typo in the URL?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1 +1,15 @@
...
+
 # Change Log
+
+All notable changes to the "azurecontainerapps" extension will be documented in this file.
```
Comment (wwlorey) [inline CHANGELOG.md:5]: IMO this line isn't required. If you'd rather keep it in though I would use the full extension name:

```suggestion
All notable changes to the "vscode-azurecontainerapps" extension will be documented in this file.
```

File: CHANGELOG.md (hunk 1 above)
Comment (nturinski) [reply CHANGELOG.md:1]: This rule is for `MD024/no-duplicate-heading`.  I could have sworn that I copied/pasted this from the Functions CHANGELOG and I thought it made sense since we have a lot of headings that are duplicated (Added, Removed, etc.) but I can't find it anymore... So I guess I'll get rid of it.

File: README.md (hunk 2 above)
Comment (nturinski) [reply README.md:5]: I believe that the url is correct.  When I click on it, it goes to the right branch/build.  Maybe it doesn't work from VS Code's markdown preview?

**Review by wwlorey — APPROVED** [review]
Looks good! Congrats on getting this baby out the door @nturinski 👏👏
